wordle-web/pages/wordEditor.vue

```diff
@@ -0,0 +1,364 @@
+  try {
+    const gameUrl = 'word/changeCommonWordFlag';
+    const headers = tokenService.value?.generateTokenHeader();
+    await Axios.post(
+      gameUrl,
+      {
+        word: word,
+        isCommonWord: isCommonWord,
+      },
+      { headers }
+    );
+  } catch (error) {
+    console.error('Error posting isCommonWord change:', error);
+  }
+}
+
+async function deleteWord(word: Word) {
+  try {
+    const gameUrl = `word/deleteWord?word=${word.word}`;
+    const headers = tokenService.value?.generateTokenHeader();
+    console.log(headers);
+    await Axios.post(gameUrl, {}, { headers });
+    let index = words.value?.indexOf(word);
+    words.value?.splice(index!, 1);
+  } catch (error) {
+    console.error('Error on deleteWord post:', error);
+  }
+}
+
+function getCommonWordIcon() {
+  switch (commonWordOrder.value) {
+    case CommonWordSortOrder.Ascending:
+      return 'mdi-chevron-up';
+    case CommonWordSortOrder.Descending:
+      return 'mdi-chevron-down';
+    case CommonWordSortOrder.Unspecified:
+      return 'mdi-unfold-more-horizontal';
+  }
+}
+
+function sortWords(words: Array<Word>) {
+  switch (commonWordOrder.value) {
+    case CommonWordSortOrder.Ascending:
+      if (isWordOrderAscending.value) {
+        words = words.sort((a, b) => {
+          let commonWordCompare =
+            Number(a.isCommonWord) - Number(b.isCommonWord);
+          if (commonWordCompare === 0) {
+            return a.word.localeCompare(b.word);
+          } else {
+            return commonWordCompare;
+          }
+        });
+      } else {
+        words = words.sort((a, b) => {
+          let commonWordCompare =
+            Number(a.isCommonWord) - Number(b.isCommonWord);
+          if (commonWordCompare === 0) {
+            return b.word.localeCompare(a.word);
+          } else {
+            return commonWordCompare;
+          }
+        });
+      }
+      break;
+    case CommonWordSortOrder.Descending:
+      if (isWordOrderAscending.value) {
+        words = words.sort((a, b) => {
+          let commonWordCompare =
+            Number(b.isCommonWord) - Number(a.isCommonWord);
+          if (commonWordCompare === 0) {
+            return a.word.localeCompare(b.word);
+          } else {
+            return commonWordCompare;
+          }
+        });
+      } else {
+        words = words.sort((a, b) => {
+          let commonWordCompare =
+            Number(b.isCommonWord) - Number(a.isCommonWord);
+          if (commonWordCompare === 0) {
+            return b.word.localeCompare(a.word);
+          } else {
+            return commonWordCompare;
+          }
+        });
+      }
+      break;
+    case CommonWordSortOrder.Unspecified:
+      if (isWordOrderAscending.value) {
+        words = words.sort((a, b) => a.word.localeCompare(b.word));
+      } else {
+        words = words.sort((a, b) => b.word.localeCompare(a.word));
+      }
+      break;
+  }
+}

```

Comment:
1. All of this can be deleted if you switch to using v-data-tables instead.

Reply:
thanks for the advise